packages/manager/modules/vrack/src/move-dialog/vrack-move-dialog.html

```diff
@@ -29,14 +29,14 @@
         <div class="oui-modal__body-text">
             <h2
                 class="oui-modal__title"
-                data-translate="vrack_move_dialog_title"
+                data-translate="{{ ::('vrack_move_dialog_title_' + $ctrl.service.type) | translate }}"
```

Comment:
You are using both directive + filter `translate`. Are they both required?

Reply:
Bad copy paste. This has been resolved. Thanks